__tests__/app.test.js

```diff
@@ -180,7 +180,122 @@ describe('POST /api/articles/:article_id/comments', ()=>{
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                article_img_url: expect.any(String),
+                votes: expect.any(Number)
+            })
+        })
+    })
+    test('200 : responds with an updated object with decremented votes given by user at the article_id', ()=>{
+        const newVotes = -10
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes:newVotes})
+        .expect(200)
+        .then(({body})=>{
+            const {article} = body
+            expect(article.votes).toBe(90)
+            expect(article).toMatchObject({
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                article_img_url: expect.any(String),
+                votes: expect.any(Number)
+            })
+        })
+    })
+    test('200 : responds with an updated object with decremented votes can be 0', ()=>{
+        const newVotes = -100
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes:newVotes})
+        .expect(200)
+        .then(({body})=>{
+            const {article} = body
+            expect(article.votes).toBe(0)
+            expect(article).toMatchObject({
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                article_img_url: expect.any(String),
+                votes: expect.any(Number)
+            })
+        })
+    })
+    test('400 : responds with an error message when inc_votes is not an number', ()=>{
+        const newVotes = 'hello'
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes:newVotes})
+        .expect(400)
+        .then(({body})=>{
+            expect(body.msg).toBe('Bad request')
+            })
+        })
+        test('400 : responds with an error message when inc_votes is higher than the votes in article', ()=>{
+            const newVotes = -200
+            return request(app)
+            .patch('/api/articles/1')
+            .send({inc_votes:newVotes})
+            .expect(400)
+            .then(({body})=>{
+                expect(body.msg).toBe('Bad request')
+                })
+            })
+            test('400 : responds with an error message when inc_votes is 0', ()=>{
+                const newVotes = 0
+                return request(app)
+                .patch('/api/articles/1')
+                .send({inc_votes:newVotes})
+                .expect(400)
+                .then(({body})=>{
+                    expect(body.msg).toBe('Bad request')
+                    })
+                })
```

Comment:
I don't think that 0 newVotes is an error case - the 400 is usually used for the wrong datatype, missing values, etc. So in this case I think the request is ok - you can just return the unchanged article with a 200